integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java

```diff
@@ -238,9 +238,21 @@ public synchronized void serverShutdown(int portOffset, boolean skipRunningCodeC
             log.info("Shutting down server ...");
 
             try {
+                /* Trying to stop the server for multiple times until it gets properly shutdown. Assuming server takes
+                around 5s to stop the server, waiting for 180s in total until the server gets shut down */
```

Comment:
moving forward this comment won't be use ful. Shall we remove it or say we wait for 3 mins.

Reply:
Changed the comment.